dist_tables/append_distribution.rst

```diff
@@ -55,38 +55,41 @@ This function informs Citus that the github_events table should be distributed b
+Citus supports two methods to load data into your append distributed tables. The first one is suitable for bulk loads from files and involves using the \\copy command. For use cases requiring smaller, incremental data loads, Citus provides two user defined functions. We describe each of the methods and their usage below.
 
-Bulk load using \\stage
+Bulk load using \\copy
 $$$$$$$$$$$$$$$$$$$$$$$
 
-The \stage command is used to copy data from a file to a distributed table while handling replication and failures automatically. It is not available in the standard `psql <http://www.postgresql.org/docs/9.5/static/app-psql.html>`_ client. Instead, Citus provides a database client called csql which supports all the commands of psql plus the \stage command.
+The `\\copy <http://www.postgresql.org/docs/current/static/app-psql.html#APP-PSQL-META-COMMANDS-COPY>`_
+command is used to copy data from a file to a distributed table while handling
+replication and failures automatically. You can also use the server side `COPY command <http://www.postgresql.org/docs/current/static/sql-copy.html>`_. 
+In the examples, we use the \copy command from psql, which sends a COPY .. FROM STDIN to the server and reads files on the client side, whereas COPY from a file would read the file on the server.
 
-Note: You can use the psql client if you do not wish to use \stage to ingest data. In upcoming releases, \stage will be replaced by more native ingest methods which will remove the need for csql.
-
-The \stage command borrows its syntax from the `client-side \copy command <http://www.postgresql.org/docs/9.5/static/app-psql.html>`_ in PostgreSQL. Behind the scenes, \stage first opens a connection to the master and fetches candidate workers on which to create new shards. Then, the command connects to these workers, creates at least one shard there, and uploads the data to the shards. The command then replicates these shards on other workers until the replication factor is satisfied and fetches statistics for these shards. Finally, the command stores the shard metadata with the master.
+You can use \\copy both on the master and from any of the workers. When using it from the worker, you need to add the master_host option. Behind the scenes, \\copy first opens a connection to the master using the provided master_host option and fetches candidate workers on which to create new shards. Then, the command connects to these workers, creates at least one shard there, and uploads the data to the shards. The command then replicates these shards on other workers until the replication factor is satisfied and fetches statistics for these shards. Finally, the command stores the shard metadata with the master.
```

Comment:
Reading the description again, it is no longer accurate for COPY:

Behind the scenes, \\copy first opens a connection to the master using the provided master_host option and uses master_create_empty_shard to create a new shard. Then, the command connects to the workers and copies data into the replicas until the size reaches shard_max_size, at which point a new shard is created. Finally, the command fetches statistics for the shards and updates the metadata.